appeals/web/src/server/appeals/appeal-details/manage-linked-appeals/manage-linked-appeals.controller.js

```diff
@@ -29,12 +29,14 @@ export const getManageLinkedAppeals = async (request, response) => {
  * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
  */
 const renderManageLinkedAppeals = async (request, response) => {
-	const { errors } = request;
-	const { appealId, parentId, relationshipId } = request.params;
-	const validId = parentId !== 'null' && parentId ? parentId : appealId;
-	const appealData = await getAppealDetailsFromId(request.apiClient, validId);
+	const {
+		errors,
+		params: { appealId }
+	} = request;
+
+	const appealData = await getAppealDetailsFromId(request.apiClient, appealId);
```

Comment:
Should this use the middleware for getting the appeal?

Reply:
Yes probably, but fixing that is outside the scope of this ticket. This was a fix to pre-existing functionality. There are many places in the codebase where `validateAppeal` could/should be used instead of getting the appeal directly, better to address all of those in one go without adding further out of scope fixes to this PR.